docs/manual/scala/guide/cluster/DomainModelling.md

```diff
@@ -1,19 +1,121 @@
 # Domain Modelling
 
+We recommend reading [[Event Sourcing and CQRS|ES_CQRS]] as a prerequisite to this section.
+
+In this section we will guide you through all the necessary steps to model an [Aggregate](https://martinfowler.com/bliki/DDD_Aggregate.html), as defined in Domain-Driven Design, using Akka Persistence Typed and following the CQRS principles.
+
+Lagom embraces Event Sourcing and CQRS by promoting the usage of Akka Persistence Typed in combination the CQRS principle. Akka Persistence Typed provides an API for building event sourced actors, but not necessarily for CQRS Aggregates. To build CQRS applications we need to apply a few rules to our design.
+
+We will explain the API and concepts while building a simplified shopping cart model. A full-fledge shopping cart sample can be found on our [samples repository](https://github.com/lagom/lagom-samples/tree/1.6.x/shopping-cart/shopping-cart-scala).
+
+You will learn, how to:
+
+* model a CQRS Aggregate using Akka Persistence Typed
+* prepare its event journal to be queryable using event tags
+* register the model on Akka's Cluster Sharding
+* look up instances on the cluster and iteract with them
+* define serializers for your commands, events and model state
+* understand the main configuation settings and their raison-être
```

Comment:
```suggestion
* understand the main configuration settings and their raison-être
```

Reply:
> raison-être

![](https://media.giphy.com/media/lE5u6gdLEXA9W/giphy.gif)

Reply:
thereis a missing d', the expressino is raison d'être :-)

Reply:
Please, add the cat-GIF to the docs.